src/js/media-volume-range.js

```diff
@@ -14,7 +14,9 @@ class MediaVolumeRange extends MediaChromeRange {
         detail: volume
       });
     });
+  }
 
+  connectedCallback () {
     this._updateRange();
```

Comment:
the diff is a little unclear, but all that happened here is that `this._updateRange();` (DOM manipulation) was moved out of the constructor and into the `connectedCallback`, which is required according to the spec (although not all browsers enforce it strictly)

https://html.spec.whatwg.org/multipage/scripting.html#custom-element-conformance

> In general, work should be deferred to connectedCallback as much as possible—especially work involving fetching resources or rendering. However, note that connectedCallback can be called more than once, so any initialization work that is truly one-time will need a guard to prevent it from running twice.